lua/astrocommunity/pack/java/README.md

```diff
@@ -43,7 +41,14 @@ We require that the root folder of your projects inlude one of these files/folde
             },
           },
         },
+        format = {
+          enabled = true,
+          settings = { -- you can use your preferred format style
+            url = os.getenv "HOME" .. ".config/nvim/formatter/eclipse-java-google-style.xml",
```

Comment:
this needs to be changed since this file is not guaranteed to exist

Reply:
```suggestion
            url = "https://raw.githubusercontent.com/google/styleguide/gh-pages/eclipse-java-google-style.xml",
```

This seems to be a good default from the `vscode-java` docs: https://github.com/redhat-developer/vscode-java/wiki/Formatter-settings

Reply:
Agree